client/src/components/About/AboutInformation/AboutInformation.js

```diff
@@ -27,53 +27,40 @@ export default class AboutInformation extends React.Component {
         const { contributors, monumentStatistics } = this.props;
         const { showingAllContributors } = this.state;
 
-        const mapNavLink = (
-            <NavLink onClick={e => {
-                e.preventDefault();
-                window.location.replace('/map');
-            }} to="/map" key="map">click here</NavLink>
+        const mapLink = (
+            <Link to="/map">click here</Link>
         );
 
         let nineElevenMemorialLink = "9/11 Memorial";
         if (monumentStatistics && monumentStatistics.nineElevenMemorialId) {
             nineElevenMemorialLink = (
-                <NavLink onClick={e => {
-                    e.preventDefault();
-                    window.location.replace(`/monuments/${monumentStatistics.nineElevenMemorialId}`);
-                }} to={`/monuments/${monumentStatistics.nineElevenMemorialId}`} key="911-memorial">9/11 Memorial</NavLink>
+                <Link to={`/monuments/${monumentStatistics.nineElevenMemorialId}`} key="911-memorial">
+                    9/11 Memorial
+                </Link>
             );
         }
 
         let vietnamVeteransMemorialLink = "Vietnam Veterans Memorial";
         if (monumentStatistics && monumentStatistics.vietnamVeteransMemorialId) {
             vietnamVeteransMemorialLink = (
-                <NavLink onClick={e => {
-                    e.preventDefault();
-                    window.location.replace(`/monuments/${monumentStatistics.vietnamVeteransMemorialId}`);
-                }} to={`/monuments/${monumentStatistics.vietnamVeteransMemorialId}`} key="vietnam-veterans-memorial">Vietnam Veterans Memorial</NavLink>
+                <Link to={`/monuments/${monumentStatistics.vietnamVeteransMemorialId}`}>Vietnam Veterans Memorial</Link>
             );
         }
 
-        let loginLink = (
-            <NavLink onClick={e => {
-                e.preventDefault();
-                window.location.replace('/login');
-            }} to="/login" key="login">click here</NavLink>
+        let suggestionLink = (
+            <Link to="/create" key="login">click here</Link>
```

Comment:
Is linking to the Create Page on purpose here?

Reply:
Nevermind, saw the changes you made in `ProtectedRoute` to redirect after they log in. Perhaps the `key` should be `create` instead of `login` then.